app/views/education_experiences/index.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= link_to t('education_experiences.new.new_experience'), new_education_experience_path, class: "button" %>
+<br>
+<br>
+<div id="page-top" class="row">
+  <div class="container-fluid home">
+    <div class="card mb-3">
+      <div class="card-header">
+        <i class="fa fa-table"></i>
+        <%= t('activerecord.models.education_experience.other') %>
+      </div>
+      <div class="card-body">
+        <div class="table-responsive">
+          <table class="table table-hover table-bordered users-evaluation" id="dataTable">
+            <thead>
+              <tr>
+                <th><%= t('activerecord.attributes.education_experience.institution') %></th>
```

Comment:
use the rails helpers to do this https://api.rubyonrails.org/classes/ActiveModel/Translation.html#method-i-human_attribute_name